src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallLibertyTask.groovy

```diff
@@ -73,6 +76,9 @@ class InstallLibertyTask extends AbstractTask {
             result.put('type', project.liberty.install.type)
         }
 
+        if(project.configurations.libertyRuntime.getFiles().size() > 1) {
```

Comment:
Can project.configurations.libertyRuntime be null here?

Reply:
This can never be null since we are creating a libertyRuntimeConfiguration in Liberty.groovy.

Reply:
Then why do we check if it is null in several other places in this class?

Reply:
I'm guessing they're not necessary, but I'm not opposed to adding the check for consistency